src/styles/main.scss

```diff
@@ -1,3 +1,42 @@
+$colors:
+  1 rgb(100%, 0%, 0%),
+  2 rgb(80%, 0%, 0%),
+  3 rgb(60%, 0%, 0%),
+  4 rgb(40%, 0%, 0%),
+  5 rgb(20%, 0%, 0%),
+  6 rgb(0%, 0%, 0%);
+
+html {
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.snake {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  &__block {
+    display: grid;
+    justify-content: center;
+    align-content: center;
+
+    height: 300px;
+    color: #fff;
+
+    @each $index, $color in $colors {
```

Comment:
you could use nth-child, and then special classes for each block wouldn't be needed

Reply:
Yeah, you're right, thanks!